crates/viewer/re_renderer/Cargo.toml

```diff
@@ -53,6 +53,7 @@ re_log.workspace = true
 re_math.workspace = true
 re_tracing.workspace = true
 re_video.workspace = true
+re_types.workspace = true
```

Comment:
We really don't want this (huge!) dependency in `re_renderer`. `re_renderer` is designed to act independent of any of the SDK types and should be able to be used without any knowledge of the database or viewer.